client-swagger/swagger.json

```diff
@@ -11787,6 +11787,38 @@
         }
       }
     },
+    "/v1/users/login" : {
+      "post" : {
+        "tags" : [ "users" ],
+        "summary" : "Login User by Password",
+        "description" : "Login a user with Password",
+        "operationId" : "loginUserWithPwd",
+        "requestBody" : {
+          "content" : {
+            "application/json" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/LoginRequest"
+              }
+            }
+          }
+        },
+        "responses" : {
+          "200" : {
+            "description" : "User Token",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                "$ref" : "#/components/schemas/LoginToken"
+                }
+              }
+            }
+          },
```

Comment:
you don't want to add e.g. "401" and/or "403" response codes.

Reply:
I would rather not touch this section of `swagger.json`, which was auto-generated